src/components/reviews/Reviews.jsx

```diff
@@ -2,15 +2,18 @@ import React from 'react';
 import ProductBreakdown from './ProductBreakdown.jsx';
 import RatingBreakdown from './RatingBreakdown.jsx';
 import ReviewsList from './ReviewsList.jsx';
+import { FilterProvider } from './FilterContext.jsx'
 import styles from './reviews.module.css';
 
 const Reviews = () => {
   return (
-    <div className={`${styles.widget} ${styles.maxSize} reviews`}>
-      <RatingBreakdown/>
-      <ReviewsList/>
-      <ProductBreakdown/>
-    </div>
+    <FilterProvider>
+      <div className={`${styles.widget} ${styles.maxSize} reviews`}>
```

Comment:
I may added an id="reviewsWidget" property to this element, so this line might cause a merge conflict cause of my previous PR merge.